main/src/com/google/refine/expr/functions/math/RandomNumber.java

```diff
@@ -64,7 +63,7 @@ public Object call(Properties bindings, Object[] args) {
                 }
             }
         }
-        return new EvalError(ControlFunctionRegistry.getFunctionName(this) + " expects two numbers, the first must be less than the second");
+        return new EvalError(ControlFunctionRegistry.getFunctionName(this) + "expects no arguments or two numbers, the first must be less than the second");
```

Comment:
I think the space before "expects" is a good thing.

Reply:
This is why I should go everywhere with my lenses 😅